reference/datasets.mdx

```diff
@@ -22,6 +28,19 @@ If you work with data sourced from different environments, separate them into di
 
 You might be tempted to use a single `environment` attribute instead, but this risks causing confusion when results show up side-by-side in query results. Although some organizations choose to collect events from all environments in one dataset, they’ll often rely on applying an `environment` filter to all queries, which becomes a chore and is error-prone for newcomers.
 
+### Separate by signal type
+
+For example, for a typical distributed application, you could split your data up as follows:
+
+- One dataset with traces for all services
+- One dataset with application logs for all services
+- One dataset with frontend web vitals
+- One dataset with infrastructure logs
+- One dataset with security logs
+- One dataset with CI logs
+
+If you are looking for some data, you probably know its signal type. But for a distributed system you might not necessarily know in advance which service is responsible for the data you want to query. This approach makes it easy to find the data that you are looking for.
+
```

Comment:
need to consider the fact that this contradicts the next section. how can we provide the best guidance?